soccer/gameplay/skills/angle_receive.py

```diff
@@ -135,6 +136,17 @@ def execute_running(self):
             main.system_state().draw_line(self._kick_line,
                                           constants.Colors.Red, "Shot")
 
+    # This depends on the superclasse's implementation somewhat
+    # If we stop using capture in pass_receive, this needs to be modified.
+    def on_enter_receiving(self):
```

Comment:
It may be a little cleaner if we have PassReceive's constructor take in a capture behavior.  That way we don't have to override this transition and replace the behavior after it's been added.